apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -1831,34 +1821,12 @@
         "administratorRoleDisplayName": "{{ console.ui.administrator_role_display_name }}",
         {% endif %}
         "legacyMode": {
-            {% if legacy_mode.enable == true %}
-            "apiResourcesV1": false,
-            "apiResourcesV2": false,
             "applicationListSystemApps": true,
-            "applicationOIDCSubjectIdentifier": true,
-            "applicationRequestPathAuthentication": true,
             "applicationSystemAppsSettings": true,
             "approvals": true,
             "certificates": true,
```

Comment:
@brionmario do we need to keep these features?

Reply:
No. Lets get rid of these

Reply:
Ack

Reply:
Removed with [de4ecd5](https://github.com/wso2/identity-apps/pull/6487/commits/de4ecd5ad82d9204c3ca5d062fc95f0eebf88c65)